app/controllers/search_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
     base = (@baseproject == "ALL") ? "" : @baseproject
 
     # if we have a baseproject, and don't show unsupported packages, shortcut: '
-    if !@baseproject.blank? && @baseproject != "ALL" && !@search_unsupported && !@search_project
+    if @baseproject.present? && @baseproject != "ALL" && !@search_unsupported && !@search_project
```

Comment:
this is not really related with moving the code to a dedicated controller... I would move it to another commit

Reply:
will do, thanks!